doc/nrf/glossary.rst

```diff
@@ -388,7 +408,7 @@ Glossary
    Repository
       The entire set of files and folders of which a project is composed, together with the revision history of each file.
       It is often composed of multiple branches.
-      It is also known as *Git repository* or *Git project*.
+      It is also known as *Git repository*, *Git project*, or *submodule*.
```

Comment:
I think `west` documentation tries not to use the term `submodule` no to mix it up with a `Git submodule` which is a bit different thing. I believe we should discuss that and choose a consistent, but non-ambiguous naming scheme.

Reply:
Let's verify and fix this in a separate PR, as this will require sifting through all the docs: https://projecttools.nordicsemi.no/jira/browse/NCSDK-11865